tools/run_tests/xds_k8s_test_driver/bin/cleanup/cleanup.py

```diff
@@ -129,7 +170,7 @@ def get_expire_timestamp() -> datetime.datetime:
     )
 
 
-def exec_gcloud(project: str, *cmds: List[str]) -> Json:
+def exec_gcloud(project: str, *cmds: str) -> Json:
```

Comment:
Should we keep`*cmds` as list?

Reply:
Nothing changed, it's just the correct way to annotate `*args`. From  https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values:

```py
def foo(*args: str, **kwds: int): ...
```

is acceptable and it means that, e.g., all of the following represent function calls with valid types of arguments:

```py
foo('a', 'b', 'c')
foo(x=1, y=2)
foo('', z=0)
```

---

```py
foo(*cmds: List[str])
```

 would mean that

```py
foo(['a', 'b'], ['c', 'd'], ...)
```

is expected